libraries/chain/db_market.cpp

```diff
@@ -52,6 +52,26 @@ namespace graphene { namespace chain { namespace detail {
  * No more asset updates may be issued.
 */
 void database::globally_settle_asset( const asset_object& mia, const price& settlement_price )
+{
+   auto maint_time = get_dynamic_global_properties().next_maintenance_time;
+   bool before_core_hardfork_1270 = ( maint_time <= HARDFORK_CORE_1270_TIME ); // call price caching issue
+
+   const auto& call_index = get_index_type<call_order_index>().indices();
+   const auto& call_price_index = call_index.get<by_price>();
+   const auto& call_collateral_index = call_index.get<by_collateral>();
```

Comment:
Don't need to look up both indexes every time, just do it inside the if blocks. No need to assign variables either.

Reply:
Updated.